CMakeLists.txt

```diff
@@ -527,7 +527,7 @@ endfunction()
 
 # The actual unit tests follow.
 #
-if(GTEST_FOUND)
+if(TARGET GTest::gtest AND TARGET GTest::gmock)
```

Comment:
Am I reading https://cmake.org/cmake/help/latest/module/FindGTest.html wrong, or is GTest::gmock only ever created starting with CMake 3.23? It doesn't look like [GTestConfig.cmake](https://github.com/google/googletest/tree/main/googletest/cmake) does anything special with targets?

Reply:
`libgtest-dev`'s `/usr/lib/x86_64-linux-gnu/cmake/GTest/GTestConfig.cmake` optionally includes `/usr/lib/x86_64-linux-gnu/cmake/GTest/GMockTargets.cmake` if present, which is the case if `libgmock-dev` is installed, which in turn adds the `GTest::gmock` library/target.

Reply:
@robinlinden THe earlier comment was for Debian Bookworm I also have them on an older Debian Bullseye system with cmake 3.18.4:

```bash
$ cmake --version
cmake version 3.18.4

CMake suite maintained and supported by Kitware (kitware.com/cmake).
$ grep -in 'add_library.*gmock' -R /usr/lib/x86_64-linux-gnu/cmake/GTest/
/usr/lib/x86_64-linux-gnu/cmake/GTest/GTestTargets.cmake:83:add_library(GTest::gmock STATIC IMPORTED)
/usr/lib/x86_64-linux-gnu/cmake/GTest/GTestTargets.cmake:92:add_library(GTest::gmock_main STATIC IMPORTED)
```

Note that these are CMake files that from gtest/gmock library packages, they are not a part of the cmake package.